add-screenshot/index.md

```diff
@@ -0,0 +1,28 @@
+## Adding screenshots to my game page
+
+There's a distinct possibility that screenshots are the single most vital element of your game pages, in terms of catching the attention of people who have already been enticed by your thumbnail, or who are visiting via a direct link to your game page. Many people will simply skip over a downloadable game with no screenshots.
+
+Even if your game is played in-browser, your game page should have some screenshots on it. A prospective player's interest might wane more quickly than the game loads, but a picture could capture their attention. And who knows, we might change the way browser games are served up in the future, so screenshots might become a gamer's only visual cue to the game's content.
```

Comment:
should we say this? "And who knows, we might change the way browser games are served up in the future, so screenshots might become a gamer's only visual cue to the game's content."